src/schema/fair.ts

```diff
@@ -47,6 +49,44 @@ const FairType = new GraphQLObjectType({
   name: "Fair",
   fields: () => ({
     ...GravityIDFields,
+    artistsConnection: {
+      type: artistConnection,
+      args: pageable({
+        section: {
+          type: GraphQLString,
+          description: "Number of artists to return",
+        },
+        sort: {
+          type: FairArtistSortsType,
+          description: "Sorts for artists in a fair",
+        },
+      }),
+      resolve: (
+        { id },
+        options,
+        _request,
+        { rootValue: { fairArtistsLoader, artistLoader } }
+      ) => {
+        const gravityOptions = omit(
+          convertConnectionArgsToGravityArgs(options),
+          ["page"]
+        )
+
+        return Promise.all([
+          totalViaLoader(fairArtistsLoader, id, gravityOptions),
+          fairArtistsLoader(id, gravityOptions),
+        ]).then(([count, { body }]) => {
+          return Promise.all(body.map(({ id }) => artistLoader(id))).then(
```

Comment:
This is less than optimal as it requests each artist individually, so you'll have as many concurrent requests in flight against gravity as the requested page size. I initially tried to get this working with `artistsLoader` but it appears the backing Gravity endpoint only accepts internal Mongo Ids: https://github.com/artsy/gravity/blob/master/app/api/v1/artists_endpoint.rb#L239-L242, as the param is passed directly to an `.in(_id: params[:ids])` filter.